### PR TITLE
Renders the countdown block in php for FSE

### PR DESCRIPTION
**Pull request by Gentyspun**
Ref: https://github.com/amnestywebsite/humanity-theme/issues/148

Reworks the countdown timer to be rendered in PHP to bring in line with FSE

**Steps to test**:
1. Edit a post
2. Countdown timer should be show in the block inserter
3. Add the block, populate and save
4. View the front end and make sure the block displays as expected

Example: http://bigbite.im/i/5f672I

### CODE REVIEW

**Review by jaymcp — DISMISSED** [review]
This needs a deprecation, so that old versions of the blocks can be transformed without breaking.

File: wp-content/themes/humanity-theme/includes/blocks/countdown-timer/render.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		$attrs = apply_filters( 'amnesty_countdown_block_attributes', $attrs );
+
+		return sprintf(
+			'<div class="clockdiv" style= text-align:%s;>
+			  <div class="countdownClock" data-timestamp=%s data-expiredText="%s" />
+			</div>',
+			esc_attr( $attrs['alignment'] ),
+			esc_attr( $attrs['date'] ),
+			esc_attr( $attrs['expiredText'] )
+		);
```
Comment (jaymcp) [inline wp-content/themes/humanity-theme/includes/blocks/countdown-timer/render.php:25]:
```suggestion
	 * @return string
	 */
	function amnesty_render_countdown_block( array $attrs = [] ): string {
		$attrs = wp_parse_args(
			$attrs,
			[
				'alignment'   => 'inherit',
				'date'        => '',
				'expiredText' => '',
			]
		);

		return sprintf(
			'<div class="clockdiv" style="text-align:%s;">
				<div class="countdownClock" data-timestamp="%s" data-expiredText="%s"></div>
			</div>',
			esc_attr( $attrs['alignment'] ),
			esc_attr( $attrs['date'] ),
			esc_attr( $attrs['expiredText'] )
		);
```

**Review by jaymcp — CHANGES_REQUESTED** [review]
Thanks for addressing the feedback. Sorry to kick this back again, but now that the deprecation's working, I can see the output of the PHP render, and I'm seeing PHP Warnings. I've left a suggested fix inline.

File: wp-content/themes/humanity-theme/includes/blocks/countdown-timer/render.php (hunk 1 above)
Comment (Gentyspun) [reply wp-content/themes/humanity-theme/includes/blocks/countdown-timer/render.php:25]: resolved

**Review by jaymcp — APPROVED** [review]
Thanks for the work on this!

**Pull request merged**